### ISSUE TITLE
exceptions.GQLException: Unauthorized: The "Authorization" token is invalid.

### ISSUE THREAD

**Issue by Kazlehoff** [DevilXD/TwitchDropsMiner#534]
Fresh install, fresh login, using latest build available date of this ticket post (Last build date: 2024-08-04 15:32:52+00:00)
```

19:14:57: Fatal error encountered:
19:14:57: 
19:14:57: Traceback (most recent call last):
19:14:57:   File "main.py", line 153, in main
19:14:57:   File "twitch.py", line 572, in run
19:14:57:   File "twitch.py", line 622, in _run
19:14:57:   File "twitch.py", line 1415, in fetch_inventory
19:14:57:   File "twitch.py", line 1362, in gql_request
19:14:57: exceptions.GQLException: Unauthorized: The "Authorization" token is invalid.
```

No Games even linked yet.

**Comment by N1Hawk** [comment]
I cannot get the miner to work, get the same error as Kazlehoff, it won't even load just gives the error above!!

**Comment by nudave** [comment]
Please see numerous threads regarding this issue. It's being worked on: 

https://github.com/DevilXD/TwitchDropsMiner/issues/526#issuecomment-2271881119

**Comment by W0lfD** [comment]
> Please see numerous threads regarding this issue. It's being worked on:
> 
> [#526 (comment)](https://github.com/DevilXD/TwitchDropsMiner/issues/526#issuecomment-2271881119)

It's another error which occurs on first request.
Looks like client_id is out of date/has been blocked by twitch. Changing it here helped me https://github.com/DevilXD/TwitchDropsMiner/blob/master/constants.py#L164

**Comment by DevilXD** [comment]
This is a duplicate of #526.
